felix/config/config_params.go

```diff
@@ -537,6 +537,10 @@ func (config *Config) TableRefreshInterval() time.Duration {
 	return config.IptablesRefreshInterval
 }
 
+func (config *Config) FlowLogEnabled() bool {
+	return config.FlowLogsGoldmaneServer != ""
```

Comment:
Is flowlogS or flowlog_ the prefered form? I'd think the plural is the desired one.